karma.conf.js

```diff
@@ -6,7 +6,7 @@ var basePath = __dirname;
 module.exports = function (config) {
   const configObject =  {
     basePath: '',
-    frameworks: ['mocha', 'chai', 'sinon-chai'],
+    frameworks: ['mocha', 'chai', 'sinon-chai', 'es6-shim'],
```

Comment:
We don't need to install es6-shim because we have installed `core-js` that provides all es6 features